pkgs/servers/code-server/default.nix

```diff
@@ -151,40 +154,80 @@ in stdenv.mkDerivation rec {
 
   buildPhase = ''
     # install code-server dependencies
-    yarn --offline
+    yarn --offline --ignore-scripts
 
-    # install vscode dependencies without running script for all vscode packages
-    # that require patching for postinstall scripts to succeed
-    for d in lib/vscode lib/vscode/build; do
-      yarn --offline --cwd $d --offline --ignore-scripts
-    done
+    # patch shebangs of everything, also cached files, as otherwise postinstall
+    # will not be able to find /usr/bin/env, as it does not exist in sandbox
```

Comment:
That is wrong. ``/usr/bin/env`` should exist in the sandbox but the binary after it might not.

Reply:
I removed the reference to `/usr/bin/env` from the comment on this, let me know if it needs additional changes.